Tools/cmake/FindDocGenerator.cmake.in

```diff
@@ -0,0 +1,17 @@
+find_package(JsonGenerator REQUIRED)
+
+function(DocGenerator)
+    set(oneValueArgs INPUT)
+
+    cmake_parse_arguments(Argument "${oneValueArgs}" ${ARGN} )
```

Comment:
I don't think this will work, cmake_parse_arguments needs 4 arguments at least:
cmake_parse_arguments(prefix options one_value_keywords multi_value_keywords args...)